packages/amplify-e2e-core/src/utils/rds.ts

```diff
@@ -126,6 +134,125 @@ export const createRDSInstance = async (
   }
 };
 
+/**
+ * Creates a new RDS Aurora serverless V2 cluster with one DB instance using the given input configuration.
+ * @param config Configuration of the database cluster. If password is not passed an RDS managed password will be created.
+ * @returns EndPoint address, port and database name of the created RDS cluster.
+ */
+export const createRDSCluster = async (
+  config: RDSConfig,
+): Promise<{
+  clusterArn: string;
+  endpoint: string;
+  port: number;
+  dbName: string;
+  password: string;
+  managedSecretArn: string;
+  dbInstance: DBInstance;
+}> => {
+  const rdsClient = new RDSClient({ region: config.region });
+  const initialDBName = 'defaultdb';
+
+  const params: CreateDBClusterMessage = {
+    /** input parameters */
+    EnableHttpEndpoint: true,
+    Engine: 'aurora-postgresql',
```

Comment:
Since we know we'll support mysql soon, let's make this an argument to the `createRDSCluster` function rather than hardcoding it. We might not want our tests to directly use the Engine value of `aurora-postgresql`, but we have a number of other specifiers for DB type / DB engine that we could use. (If this wasn't a requirement we knew was coming soon, I'd agree with your hardcoded approach--[YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it)!)